README.md

```diff
@@ -59,10 +59,12 @@ var css = fs.readFileSync("input.css", "utf8")
 
 ### Configuring rules
 
-[Rules](docs/rules.md) are configured within the `rules` key of the config. Each rule can be turned off or on:
+[Rules](docs/rules.md) are configured within the `rules` key of the config.
+Like [ESLint](http://eslint.org/docs/user-guide/configuring#configuring-rules), each rule can be turned off or on:
 
 * 0 - turn the rule off
-* 2 - turn the rule on
+* ~~1 - turn the rule on as a warning (doesn't affect exit code)~~ @todo
+* 2 - turn the rule on as an error (~~exit code is 1 when triggered~~ might affect exit code to 1 in a near future)
```

Comment:
How's about, to keep things consistent, using the same approach as the inaccurate line numbers? e.g. 

* 0 - turn the rule off	
* 2 - turn the rule on

*Note: see issue #26 for details of our progress towards adding severities i.e. "1 - turn rule on as a warning & 2 - turn rule on as an error"*.